pkg/apis/certmanager/v1alpha1/types_issuer.go

```diff
@@ -182,6 +188,18 @@ type VaultAppRole struct {
 	SecretRef SecretKeySelector `json:"secretRef"`
 }
 
+type KubernetesAuth struct {
+	// The Kubernetes auth backend mount point if different from the default "kubernetes"
+	// +optional
+	Path string `json:"mountPath"`
+
+	// Secret containing a Kubernetes ServiceAccount JWT used for authenticating with Vault
+	SecretRef SecretKeySelector `json:"secretRef"`
```

Comment:
This field is required? It might be worth explicitly calling it out here, and mentioning that 'ambient credentials' (i.e. using cert-manager's own service account) is not supported here.